itests/splitstore_test.go

```diff
@@ -477,7 +476,7 @@ func (g *Garbager) createMiner(ctx context.Context) *lapi.MsgLookup {
 	createStorageMinerMsg := &types.Message{
 		To:    power.Address,
 		From:  worker,
-		Value: big.Zero(),
+		Value: types.FromFil(1),
```

Comment:
Calculating deposit from api is probably more stable so there's less chance of this flaking.  API is at `g.node`.